diff --git a/plugins/availability/alma.inc b/plugins/availability/alma.inc
index 3db7790..4769a2b 100644
--- a/plugins/availability/alma.inc
+++ b/plugins/availability/alma.inc
@@ -93,7 +93,8 @@ function alma_availability_periodical($provider_ids) {
 
   $holding_parts = array('branch', 'department', 'location', 'sublocation', 'collection');
   $details = alma_client_invoke('catalogue_record_detail', $ids);
-  $reservable_statuses = array('availableForLoan', 'checkedOut');
+  $reservable_statuses = array('availableForLoan', 'checkedOut', 'periodical', 'inAcquisition', 'ordered', 'openLoan');
+
 
   $result = array();
   if ($details && isset($details['records'])) {
